source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -21,6 +21,7 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "BioXASImagingAppController.h"
 
+#include "beamline/CLS/CLSBeamlines.h"
```

Comment:
I can see what you're trying to do here Sheldon. I think it's a good idea. I'm not a big fan of the nomenclature for the QStrings though, it really should be camel case and fully worded:
`SGMBeamlineName` rather than `SGM_bl_name`
Also, I would say that it really should be a function of the namespace, something that takes one of the enums and returns the correct string.
`QString beamlineName(CLSBeamline::CLSBeamlines beamline)`